sanic/app.py

```diff
@@ -92,6 +87,18 @@ def __init__(
         self.go_fast = self.run
         self.test_mode = False
 
+        # Signal Handlers and Registry
+        self._signals = {
+            "server": Namespace(namespace="server", owner=self, classic_event=True,),
+            "request": Namespace(namespace="request", owner=self),
+            "response": Namespace(namespace="response", owner=self),
+            "middleware": Namespace(namespace="middleware", owner=self),
+        }
```

Comment:
Looks like 4 Namespaces of signals are defined, but signal emitters are only implemented for `server` signals. Are the others planned to be implemented?